source/visionEnhancementProviders/screenCurtain.py

```diff
@@ -80,13 +80,17 @@ def canStart(cls):
 	def __init__(self):
 		super(VisionEnhancementProvider, self).__init__()
 		Magnification.MagInitialize()
-		Magnification.MagShowSystemCursor(False)
 		Magnification.MagSetFullscreenColorEffect(TRANSFORM_BLACK)
+		Magnification.MagShowSystemCursor(False)
 
 	def terminate(self):
-		super(VisionEnhancementProvider, self).terminate()
-		Magnification.MagShowSystemCursor(True)
-		Magnification.MagUninitialize()
+		try:
+			super(VisionEnhancementProvider, self).terminate()
+			Magnification.MagUninitialize()
+		except:  # noqa: E722

```

Comment:
These two lines don't have to be there. not specifying an except in a try/finally will just raise.

Having said that, the only exception I expect is Windows Error.